src/MySqlConnector/Core/Row.cs

```diff
@@ -241,6 +241,8 @@ public int GetInt32(int ordinal)
 
 	protected abstract int GetInt32Core(ReadOnlySpan<byte> data, ColumnDefinitionPayload columnDefinition);
 
+	protected abstract MySqlDecimal GetMySqlDecimalAsStringCore(ReadOnlySpan<byte> data, ColumnDefinitionPayload columnDefinition);
```

Comment:
When is this method used?

Reply:
whenever the user have data type.
I have only added this because you asked me to in earlier comments, btw its same as method above GetInt32core maybe we want to keep the consistancy
i can remove it if you want me to

Reply:
I didn't ask for it; I merely suggested that "[you] will likely need" such a method based on how your code looked at the time. Since then, you've changed the code.

But as far as I can tell, this method is never invoked, so it's dead code. If this method is never called, please delete it. (And if it is called, I must have missed it.)

Reply:
oh okay, sorry for the confusion, I will delete it, and thank you for your suggestions